Vagrantfile

```diff
@@ -73,6 +73,10 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         # your development guest becoming corrupted (in which case you should only need to do a
         # vagrant destroy and vagrant up to get a new one).
         domain.volume_cache = "unsafe"
+
+        # Bodhi should not use user sessions since it requires network customization
```

Comment:
Is the network customization the port forwarding we do?

Reply:
That is what I've understood from Vit reply. It makes sense to me that a user session cannot forward ports to the host system.